test/test_restapi.py

```diff
@@ -1162,6 +1162,99 @@ def test_vrf_routes_all_verbs(self, setup_restapi_client):
         j = json.loads(r.text)
         assert sorted(j) == sorted(routes)
 
+    def test_vrf_persistent_routes_all_verbs(self, setup_restapi_client):
+        _, _, _, restapi_client = setup_restapi_client
+        routes = []
+        routes.append({'cmd':'add',
+                            'ip_prefix':'20.1.2.0/24',
+                            'nexthop':'192.168.2.200',
+                            'persistent': 'true'})
+
+        routes.append({'cmd':'add',
+                            'ip_prefix':'30.1.2.0/24',
+                            'nexthop':'192.168.2.200,192.168.2.201',
+                            'persistent': 'true'})
+
+        routes.append({'cmd':'add',
+                            'ip_prefix':'40.1.2.0/24',
+                            'nexthop':'192.168.2.200,192.168.2.201,192.168.2.202',
+                            'ifname':'Ethernet0,Ethernet4,Ethernet8',
+                            'persistent': 'true'})
+
+        routes.append({'cmd':'add',
+                            'ip_prefix':'50.1.2.0/24',
+                            'ifname':'Ethernet0,Ethernet4',
+                            'persistent': 'true'})
+
+        routes.append({'cmd':'add',
+                            'ip_prefix':'60.1.2.0/24',
+                            'ifname':'Ethernet8',
+                            'persistent': 'true'})
+
+        routes.append({'cmd':'add',
+                            'ip_prefix':'70.1.2.0/24',
+                            'nexthop':'192.168.2.200,192.168.2.201,192.168.2.202',
+                            'nexthop_monitor':'192.168.3.200,192.168.3.201,192.168.3.202',
+                            'weight':'10,20',
+                            'profile':'profile1',
+                            'persistent': 'true'})
+
+
+        # Patch add
```

Comment:
Suggest please add a unit-test for refresh field. Also, please have one or two cases with IPv6 entries as that is the main use-case